README.md

```diff
@@ -86,7 +86,7 @@ The conditions are AND-ed.  Regexes are more resource intensive and hence should
 
 * All incoming matrics are validated and go into the table when valid.
 * The table will then check metrics against the blacklist and discard when appropriate.
-* Then metrics pass through the rewriters and are modified if applicable.
+* Then metrics pass through the rewriters and are modified if applicable.  Rewrite rules wrapped with forward slashes are interpreted as regular expressions.
```

Comment:
there's 2 places in the code that show `addRewriter <old> <new> <max>.*add rewriter that will rewrite all old to new, max times`
can you update those and mention that `/old/` will apply it as a regex